app/services/gmail/unread.py

```
@@ -0,0 +1,326 @@
+                "email": "",
+                "subjects": [],
+                "first_date": None,
+                "last_date": None,
+                "message_ids": [],
+                "total_size": 0,
+            }
+        )
+        processed = 0
+        batch_size = 100
+
+        def process_message(request_id, response, exception) -> None:
```

Ruff (0.14.10)

Unused function argument: `request_id`

(ARG001)